plugin/core/sessions.py

```diff
@@ -509,16 +547,15 @@ def should_notify_did_change_configuration(self) -> bool:
         return self.has_capability("didChangeConfigurationProvider")
 
     def handles_path(self, file_path: Optional[str]) -> bool:
+        if self._supports_workspace_folders():
+            return True
```

Comment:
So this allows running sessions on files outside the workspace... Sneaky sneaky. ;)

Although there is an issue. I've just opened an external `js` file in an LSP project and `LSP-eslint` went bananas - initializing and closing itself in an infinite loop.

Reply:
That’s a bug that I will have to work on.

Reply:
So for this PR maybe don't enable starting sessions for files outside of the workspace?

Reply:
Done.